services/vaults/apy/save/handler.js

```diff
@@ -276,7 +276,7 @@ const readVault = async (vault) => {
   return data;
 };
 
-module.exports.handler = async () => {
+const handler = async () => {
```

Comment:
why change this? was it broken?

Reply:
At the end of the file I have set:
module.exports = { handler, getVirtualPrice };

The reason is that I'm reusing the same getVirtualPrice function in : https://github.com/miguel567/yearn-api/blob/91c1696f5f5fa309126883fd17a51ee964a1448b/services/vaults/holdings/save/getHoldings.js#L94

so I had to modify the exports module to include both functions.